tsconfig.test.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
- [ ] can you please confirm that all is good with syntax highlighting in the test folder (and the regular folder)?

Reply:
syntax highlighting works correctly